backends/gstreamer/audio_stream_reader.rs

```diff
@@ -32,12 +32,12 @@ impl GStreamerAudioStreamReader {
         // gstreamer bug
         let caps = Caps::new_simple("audio/x-raw", &[("layout", &"interleaved")]);
```

Comment:
Ideally you'd use `gst::Caps::builder()` everywhere (and `gst::Structure::builder()`, etc). Doesn't this cause a deprecation warning?

Reply:
Yes, it does cause a deprecation warning. I didn't fix yet all the warnings and unused imports yet. I was focusing first on fixing the errors and making it all compile with the latest version.

Reply:
Done.